adm/_make_and_install_crontab.sh

```diff
@@ -19,10 +19,9 @@ echo_ok
 
 echo -n "- Installing crontab file..."
 echo_running
-deploycron_file "${MODULE_RUNTIME_HOME}/tmp/config_auto/crontab"
+deploycron_file "${MODULE_RUNTIME_HOME}/tmp/config_auto/crontab" > /dev/null 2>&1
 if test $? -eq 0; then
     echo_ok
 else
-    echo_nok
-    exit 1
+    echo_warning "Can't install crontab file (normal if you are in continuous integration with docker)"
```

Comment:
This line is too long. I propose instead:
```bash
echo_nok "(can't install)"
```

I propose `echo_nok` and not `echo_warning` because of test introduced in previous comment.


```